waspc/data/Generator/templates/server/src/auth/providers/config/local.ts

```diff
@@ -0,0 +1,22 @@
+{{={= =}=}}
```

Comment:
This is a generic oAuth provider config file. When we need a config for Google, we copy base it on this file and name the file `google.ts`.